stacks/README.md

```diff
@@ -3,3 +3,11 @@
 ### C or C++
 
 1. [Balanced Parenthesis](c-or-cpp/balanced-parenthesis.cpp)
+
+### Java 
+
+1. [Stack Implementation](JAVA/stack.java)
+2. [Balanced Parenthesis](JAVA/Balanced_Paranthesis.java)
+3. [Stock Span Problem](JAVA/The_Stock_Span_Problem.java)
+4. [Main.java](JAVA/Main.java) - just for testing
```

Comment:
Please @sridhar-5 remove the main class from here

You can add main function in each program like this example [here](https://github.com/MakeContributions/DSA/blob/main/scheduling/java/round-robin.java)

Reply:
Will Update you soon on this.

Reply:
All the suggestions have been addressed. can you please review it now?